superset/business_type/internet_address.py

```diff
@@ -0,0 +1,138 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This file (and `port.py`) should preferably be placed in a sub-package like `plugins` or similar to distinguish them from the other more generic code here.